### ISSUE TITLE
Implement auth for client side

### ISSUE THREAD

**Issue by sujin-lifology** [sujin-lifology/project_pilot#4]

**Comment by henriquebuzon** [comment]
Can you provide some more detail?

It seems that there is no authentication on the server side either, not even a API endpoint for authentication purposes, right?

I don't know how I can implement the auth for the client side without these features. 

I can create the authentication on the server side too if you want, but for that I will need to establish a database connection and install an ORM.

**Comment by sujin-lifology** [comment]
Yeah, you can work on the server and client side. Use JWT tokens for authentication, and for the database, use PostgreSQL run it on the local server and create just users table that will be enough i will update the changes of database in the upcoming days. I just started this project a couple of days ago, and I have an exam the day after tomorrow. So, after that, I will be working on this too. If you have any doubts, DM me on Twitter (link in my bio). Fork the repository and create a new branch to work on. After completion, make a pull request, and I will merge it into the main branch! Thanks for your comment.
